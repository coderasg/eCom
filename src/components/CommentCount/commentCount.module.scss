.commentCount {
  display: flex;
  align-items: center;
}
