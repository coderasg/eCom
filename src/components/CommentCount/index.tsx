import React from 'react';
import CommentIcon from './../../components/Icons/CommentIcon/index.svg';
import { $iconDefaultColor } from './../../components/Icons/IconColors';

import styles  from './commentCount.module.scss';

type Props = {
  count: number;
};

function CommentCount(props: Props) {
  const { count } = props;
  return <div data-testid="CommentCount-wrapper" className={styles.commentCount}>
    <CommentIcon width={25} height={25} fill={$iconDefaultColor} />
    {`${count}`}
  </div>;
}

export default CommentCount;
