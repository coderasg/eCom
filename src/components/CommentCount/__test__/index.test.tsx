import React from 'react';
import renderer from 'react-test-renderer';

import CommentCount from '../index';

describe('(Component) CommentCount', () => {
  const defaultProps = {
    count: 1,
  };

  test('should pass snapshot test', () => {
    const wrapper = renderer.create(<CommentCount {...defaultProps} />);
    expect(wrapper.toJSON()).toMatchSnapshot();
  });
});
