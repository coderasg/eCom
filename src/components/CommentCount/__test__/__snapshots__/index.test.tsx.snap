// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`(Component) CommentCount should pass snapshot test 1`] = `
<div
  data-testid="CommentCount-wrapper"
>
  <SvgIcon
    fill="#b1b0b5"
    height={25}
    width={25}
  />
  1
</div>
`;
